SPEC.md

```
@@ -1,29 +1,275 @@
+| Language | Go 1.24 |
+| Database | SQLite (with golang-migrate) |
+| Templates | templ |
+| Styling | Tailwind CSS |
+| Build | Taskfile |
+| Scheduler | gocron v2 |
+| Logging | zerolog |
+| SQL Generation | sqlc |
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)